pkg/scheduler/internal/cache/interface.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package cache
 
 import (
-	v1 "k8s.io/api/core/v1"
```

Comment:
nit: please revert this line.

Reply:
Done, but there should be no need for that name.